package.json

```diff
@@ -111,6 +115,12 @@
     "test": "jest --coverage",
     "watch": "jest --watch"
   },
+  "lint-staged": {
+    "*.js": [
+      "prettier --write",
```

Comment:
@ybiquitous what about [pretty-quick](https://github.com/prettier/pretty-quick) instead?

Reply:
@Mouvedia We use `lint-staged`, isn't pretty-quick unnecessary?